agent-control/src/cli/on_host/host_monitoring_gen/otel_config_gen.rs

```diff
@@ -0,0 +1,113 @@
+use crate::cli::error::CliError;
+use std::path::PathBuf;
+use tracing::info;
+
+/// Represents the values to create or migrate an infra-config
+pub struct OtelConfigGen {
+    otel_agent_values_path: PathBuf,
+    otel_config_source_path: PathBuf,
+}
+
+impl Default for OtelConfigGen {
+    fn default() -> Self {
+        Self {
+            otel_agent_values_path: PathBuf::from(
+                "/etc/newrelic-agent-control/fleet/agents.d/nr-otel-collector/values",
+            ),
+            otel_config_source_path: PathBuf::from(
+                "/etc/newrelic-agent-control/examples/values-nr-otel-collector-agent-linux.yaml",
+            ),
+        }
+    }
+}
+
+impl OtelConfigGen {
+    #[allow(dead_code)]
+    fn new(otel_agent_values_path: &str, otel_config_source_path: &str) -> OtelConfigGen {
+        Self {
+            otel_agent_values_path: PathBuf::from(otel_agent_values_path),
+            otel_config_source_path: PathBuf::from(otel_config_source_path),
+        }
+    }
```

Comment:
If not used, I'd get rid of it instead of allowing the dead code.

Reply:
Yes, it's done in last commit